src/ThreeEditor/components/Sidebar/SidebarTree/SidebarTree.tsx

```diff
@@ -79,6 +80,24 @@ export function SidebarTree(props: { editor: Editor; sources: TreeSource }) {
 
 	const objectRefs = useRef<Map<string, HTMLDivElement>>(new Map());
 
+	const canDrop = (
+		object: TreeItem | undefined,
+		target: TreeItem | undefined,
+		dropTargetId: string | number
+	) => {
+		if (!object) return false;
+
+		const object3d = object.data?.object;
+
+		if (isQuantity(object3d)) return object3d.parent === target?.data?.object;
+
+		if (isOutput(object3d)) {
+			return object.parent === dropTargetId;
+		}
```

Comment:
Why don't you use consistent coding style here ?

Reply:
Good point